projects/frontend/cicd/install_shared.sh

```diff
@@ -5,7 +5,7 @@
 echo "Logging npm engines version..."
 npm version
 echo "Installing dependencies..."
-npm install
+npm install --max_old_space_size=1000
```

Comment:
@gorankokin 
You want me to put 
```
node --max-old-space-size=1000 `which npm` install
```
Here?

Reply:
yes